app/organisation/rest.py

```diff
@@ -135,7 +135,7 @@ def get_organisation_services_usage(organisation_id):
         return jsonify(result='error', message='No valid year provided'), 400
     services = fetch_usage_year_for_organisation(organisation_id, year)
     list_services = services.values()
-    sorted_services = sorted(list_services, key=lambda s: s['service_name'].lower())
+    sorted_services = sorted(list_services, key=lambda s: (-s['active'], s['service_name'].lower()))
```

Comment:
why is the dash there? Sorry if that's a silly question 😅

Reply:
```python
-True
>>> -1

-False
>>> 0
```
😨 I have learned something new about Python today

Reply:
The dash reverses the order, so I get True before False

Reply:
sweet, thanks a lot, I had no idea this is a thing, today I've learnt :3